### PR TITLE
Swap database from Mysql to Postgres

### PR DESCRIPTION
**Pull request by hunj**
https://stackoverflow.com/a/585639

This will make it easier to deploy later down the road, because Django+Postgresql is more common than Django+mysql.
Also, Postgres is smart enough to read environment variables when the docker image is initially built -- easier onboarding :sparkles:

### CODE REVIEW

**Review by h4ck3r-ky13 — APPROVED** [review]
lgtm

**Comment by h4ck3r-ky13** [comment]
@dquach217

**Comment by dquach217** [comment]
Other than that one comment made, everything else lgtm

**Comment by h4ck3r-ky13** [comment]
> Other than that one comment made, everything else lgtm

I don't see a comment from you in this thread. I am also mobile sooo idk

File: docker-compose.yml [hunk 1]
```diff
@@ -26,16 +26,16 @@ services:
...
+  # nginx:
+  #   container_name: nginx
+  #   image: nginx:latest
+  #   ports:
+  #     - 80:8080
+  #   volumes:
+  #     - ./nginx/nginx-proxy.conf:/etc/nginx/conf.d/default.conf:ro
+  #     - ./web/build:/var/www/frontend # maps frontend build inside nginx
+  #   depends_on:
+  #     - django
```
Comment (dquach217) [inline docker-compose.yml:39]: Is there a reason we are keeping this here when it's being commented out? Are there plans to reintroduce this code snippet?

File: .gitignore [hunk 2]
```diff
@@ -170,6 +170,7 @@ dmypy.json
 
 secrets/
 data/mysql
```
Comment (dquach217) [inline .gitignore:172]: since we are migrating from mysql to postgres, not sure if we really need this here. There is no harm in keeping it.

**Review by dquach217 — COMMENTED** [review]
@hunj could you dm the answers to one of the comments made here? The other one is really negligible.

**Comment by h4ck3r-ky13** [comment]
ty david

File: .gitignore (hunk 2 above)
Comment (hunj) [reply .gitignore:172]: Good point, we can scrap this

File: docker-compose.yml (hunk 1 above)
Comment (hunj) [reply docker-compose.yml:39]: We can remove them for sure!

**Pull request merged**